presto-main/src/test/java/com/facebook/presto/execution/TestQueryStats.java

```diff
@@ -84,6 +49,101 @@ public void testJson()
         assertExpectedQueryStats(actual);
     }
 
+    @Test
```

Comment:
There's no need to add another test for just one set of properties - reuse `assertExpectedQueryStats`. Revert the other changes.